packages/DataGrid/src/DataGrid.js

```diff
@@ -0,0 +1,628 @@
+                  a11yText={a11yText}
+                  column={column}
+                  columnIndex={columnIndex}
+                  data={data}
+                  gridId={gridId}
+                  ref={handleRefCell({ columnIndex, rowIndex })}
+                  rowIndex={rowIndex}
+                  style={style}
+                />
+              );
+            }}
+          </Grid>
+
+          {/** COLUMS */}
+
+          <Grid
+            className={`grid-${gridId}`}
+            columnCount={columnCount}
+            columnWidth={columnIndex => {
+              if (stickyColumnsIndexes.includes(columnIndex)) return 0;
+              return ColumnDefinitions[columnIndex].props.width;
+            }}
+            height={calculatedTableHeight}
+            innerElementType={innerElementTypeMainGrid}
+            onItemsRendered={handleItemsRedered}
+            onScroll={handleScroll}
+            outerElementType={outerElementTypeMainGrid}
+            overscanColumnCount={overscanColumnCount}
+            overscanRowCount={overscanRowCount}
+            ref={refGrid}
+            rowCount={rowCount}
+            rowHeight={() => rowHeight}
+            width={gridWidth - stickyGridWidth}
+          >
+            {({ columnIndex, rowIndex, style }) => {
+              const column = ColumnDefinitions[columnIndex].props;
+              const cellA11yText =
+                typeof column.cell === "function"
+                  ? column.cellA11yText && column.cellA11yText({ row: data[rowIndex], rowIndex, columnIndex })
+                  : data[rowIndex][column.cell];
+              const headerA11yText = columnHeadersA11yText[columnIndex];
+              const a11yText = a11yTextMessage(cellA11yText, headerA11yText, rowIndex);
+
+              if (stickyColumnsIndexes.includes(columnIndex)) {
+                return null;
+              }
+
+              return (
+                <Cell
+                  a11yText={a11yText}
+                  column={column}
+                  columnIndex={columnIndex}
+                  data={data}
+                  gridId={gridId}
+                  ref={handleRefCell({ columnIndex, rowIndex })}
+                  rowIndex={rowIndex}
+                  style={style}
+                />
+              );
+            }}
+          </Grid>
+        </styled.Flex>
+        <styled.FillerTopRigth rowHeight={rowHeight} scrollBarWidth={scrollBarWidth} />
+        <styled.FillerBottomLeft stickyGridWidth={stickyGridWidth} scrollBarWidth={scrollBarWidth} />
+      </styled.Grid>
+      {!isIdle && !hasNoRecords ? (
+        <>
+          <styled.Footer $width={gridWidth}>
+            <styled.RowCount>
+              Rows:{rowCount} Columns:{columnCount}
+            </styled.RowCount>
+          </styled.Footer>
+          {WhenScrollBarReachedBottom ? (
+            <End width={gridWidth} ref={refEnd}>
+              {WhenScrollBarReachedBottom}
+            </End>
+          ) : null}
```

Comment:
Could get rid of these two `nulls` by doing `{whatever &&...` instead of `{whatever ?...`